src/uct/ib/base/ib_md.c

```diff
@@ -1080,29 +1107,24 @@ ucs_status_t uct_ib_md_open(uct_component_t *component, const char *md_name,
         uct_ib_fork_warn_enable();
     }
 
-    for (i = 0; i < ucs_static_array_size(uct_ib_ops); i++) {
-        status = uct_ib_ops[i]->ops->open(ib_device, md_config, &md);
-        if (status == UCS_OK) {
-            ucs_debug("%s: md open by '%s' is successful", md_name,
-                      uct_ib_ops[i]->name);
-            break;
-        } else if (status != UCS_ERR_UNSUPPORTED) {
-            goto out_free_dev_list;
-        }
-        ucs_debug("%s: md open by '%s' failed, trying next", md_name,
-                  uct_ib_ops[i]->name);
+    if (component == &uct_ib_component) {
+        status = uct_ib_component_md_open(ib_device, md_config, md_name, &md);
+    } else {
+        status = uct_ib_mlx5_gga_md_open(ib_device, md_config, &md);
```

Comment:
can we call this function directly from gga code (so gga will not point to uct_ib_md_open, but to its own func)?